ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,35 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=None,
+        shape=None,
+        min_axis=None,
+        max_axis=2,
```

Comment:
To ensure the array is 2D you will need to set the `min_num_dims` and `max_num_dims`, not the `min_axis` and `max_axis`. Please read the docstring of `dtype_values_axis` carefully to understand its arguments.